mlrun/model_monitoring/applications/base.py

```diff
@@ -145,6 +146,35 @@ def call_do_tracking(event: Optional[dict] = None):
         else:
             return call_do_tracking()
 
+    @staticmethod
+    def _handle_endpoints_type(project: str, endpoints) -> list[tuple[str, str]]:
+        if endpoints:
+            if isinstance(endpoints, str) or (
+                isinstance(endpoints, list) and isinstance(endpoints[0], str)
+            ):
+                endpoints_list = (
+                    mlrun.get_run_db()
+                    .list_model_endpoints(project, names=endpoints, latest_only=True)
+                    .endpoints
+                )
+                if endpoints_list:
+                    endpoints = [
+                        (endpoint.metadata.name, endpoint.metadata.uid)
+                        for endpoint in endpoints_list
+                    ]
+                else:
+                    raise mlrun.errors.MLRunNotFoundError(
+                        f"Did not find any model endpoint named ' {endpoints}'"
+                    )
```

Comment:
I think you should check that the len of model endpoints are the same as the users list, and if not to raise this exception